libs/database/src/lib/repositories/index.ts

```diff
@@ -1360,4 +1359,13 @@ export class Repository {
       this.getWalletShortcutQuery(filter),
     );
   }
+
+  async getUserAvailable(suggestCastcleId: string) {
```

Comment:
```suggestion
  async suggestCastcleId(...preferKeywords: string[]) {
```
ควรรับพวก keyword ที่อยากตั้ง เช่น preferCastcleId, firstName, lastName, socialName อะไรแบบนี้

ถ้ามี any keyword ก็เอามาตั้ง ถ้าซ้ำก็เติมเลข (number)
ถ้าไม่มี ก็ค่อยไปเอาพวก keywords จาก s3 (+number ถ้าซ้ำ)

ปล. ไม่อยากให้ใช้ `Date.now()` เพราะมันมีโอกาสที่มีคนตั้งล่วงหน้าไว้ แล้วมันไปซ้ำ